training/general/src/environments/pytorch-1.11-cuda11.3/context/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM mcr.microsoft.com/azureml/aifx/stable-ubuntu2004-cu113-py38-torch1110:{{lat
 
 # Install pip dependencies
 COPY requirements.txt .
-RUN pip install -r requirements.txt
+RUN pip install -r requirements.txt --no-cache-dir
 
 # Inference requirements
 COPY --from=mcr.microsoft.com/azureml/o16n-base/python-assets:20220607.v1 /artifacts /var/
```

Comment:
Is there a newer version of python-assets? I think I asked this before but I can't find the comment.

Reply:
I'm not sure, this environment was added recently to AzureMLCli and it has this version, so I just added the same DockerFile being used there. Its probably good to have the environments match in both places until we remove them from AzureMLCli though

Reply:
Okay, maybe something to think about for future PRs.